numba/compiler_machinery.py

```diff
@@ -0,0 +1,375 @@
+from __future__ import print_function, absolute_import
+
+import time
+from abc import ABC, abstractmethod
+from collections import namedtuple, OrderedDict
+import inspect
+import traceback
+from numba.compiler_lock import global_compiler_lock
+from numba import errors
+from . import config, utils, transforms
+from .tracing import event
+from .postproc import PostProcessor
+
+# terminal color markup
+_termcolor = errors.termcolor()
+
+
+class SimpleTimer():
+    """
+    A simple context managed timer
+    """
+
+    def __enter__(self):
+        self.ts = time.time()
```

Comment:
Use `timeit.default_timer` for better cross platform timer precision

Reply:
thanks. fixed.